src/atoms/progress-bar/progress-bar.test.tsx

```diff
@@ -1,5 +1,104 @@
 import React from "react";
+import faker from "faker";
+import {
+    ProgressBar,
+    ProgressBarStyles,
+    ProgressBarErrorClass,
+} from "./progress-bar";
+import { render, getByTestId } from "@testing-library/react";
 
 describe("ProgressBar", () => {
-    test.skip("TODO - https://github.com/AndcultureCode/AndcultureCode.JavaScript.React.Components/issues/13", () => {});
+    test("when default props, renders progress bar", () => {
+        // Arrange
+        const expected = "dataTestId";
```

Comment:
Usually we use the name `expected` when setting up a value that we're asserting on after the act step. Being as the act step doesn't calculate or modify anything related to this variable, and we're not actually asserting on its value, I would probably rename this to just `testId`